doc/release-notes/TypeScript 3.7.md

```diff
@@ -0,0 +1,848 @@
+# TypeScript 3.7
+
+## 可选链（Optional Chining）
```

Comment:
Typo: should be `Chaining`

Reply:
fixed (and some other stuff)
but I saw link problems in `SUMMARY.md` `preface.md` `README.md` while checking links

Should I make a quick fix for these three files just in this PR or later

Reply:
Let's merge the main content first. The links are not big problemt. :D